### PR TITLE
Modules - Stringtabled

### PR DESCRIPTION
**Pull request by gienkov**
Done:
* Advanced Ballistics module
* Ballistics - ammo crate
* Captives module
* Common module
* 2 Vehiclelock modules
* Weather module
* Winddeflection module
* Hearing module
* Interaction module
* Map and BFT modules
* SwitchUnits module
* Ambiance Sounds - module, category
* Medical - all 6 modules, 2 chests, editor category, fixes to CMS leftovers in descriptions

Also:
* Added module descriptions

Left:
* Allow Config Export [ACE] - module
* Explosive System - module
* Friendly Fire Messages - module
* MicroDAGR Map Fill - module
* MK6 Settings - module
* Name Tags
* Rallypoint System
* Respawn System

### CODE REVIEW

**Comment by KoffeinFlummi** [comment]
Going to put this down as 3.1.0 for now.

**Comment by gienkov** [comment]
Will finish this till sunday I hope

File: addons/captives/config.cpp [hunk 1]
```diff
@@ -6,7 +6,7 @@ class CfgPatches {
         weapons[] = {"ACE_CableTie"};
         requiredVersion = REQUIRED_VERSION;
         requiredAddons[] = {"ACE_Interaction"};
-        author[] = {"commy2", "KoffeinFlummi"};
+        author[] = {"commy2", "KoffeinFlummi", "GieNkoV"};
```
Comment (esteldunedain) [inline addons/captives/config.cpp]: I don't think adding stringtables is quite enough to list yourself as author of the module. @PabstMirror rewritted 3/4 of this module for ACE and it's not even listed here! (btw, he should be).

The same is true for most modules, which at this point have been extensively rewritten/ported/ammended by core developers not listed as authors. 

We may want to change to something like `author[] = {"ACE Team"};` at some point

**Comment by bux578** [comment]
> We may want to change to something like author[] = {"ACE Team"}; at some point

:+1:

File: addons/captives/config.cpp (hunk 1 above)
Comment (gienkov) [inline addons/captives/config.cpp]: No problem then, it taken me few hours already to do this

**Comment by gienkov** [comment]
You can always change that string names if they are too long or something, Im doing this for the first time and I don't know how it should look exactly.

**Comment by gienkov** [comment]
Is it ok already?

**Pull request closed without merging**